raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -325,8 +341,30 @@ private void fireHandleResign() {
     }
 
     @Override
-    public void initialize() throws IOException {
-        quorum.initialize(new OffsetAndEpoch(log.endOffset().offset, log.lastFetchedEpoch()));
+    public void initialize(String quorumVoterStrings) throws IOException {
+        List<Node> quorumVoterNodes = quorumVoterStringsToNodes(quorumVoterStrings);
+        Set<Integer> quorumVoterIds = quorumVoterNodes.stream().map(Node::id).collect(Collectors.toSet());
```

Comment:
This is the same as `voterAddresses.keySet()` which is computed below.

Reply:
Ack. Addressing these in the changes. I am moving all computation of voter nodes/ids into RaftConfig. That should simplify this and avoid double work